docs/cep/query-guide/functions/sinkmapper/text.md

```diff
@@ -2,69 +2,93 @@
-| delimiter              | This parameter specifies how events are separated when a grouped event is received. This must be a whole line and not a single character.                                                                                                                                     | \~\~\~\~\~\~\~\~\~\~ | STRING              | Yes      | No      |
-| new.line.character     | This attribute indicates the new line character of the event that is expected to be received. This is used mostly when communication between 2 types of operating systems is expected. For example, Linux uses `\n` whereas Windows uses `\r\n` as the end of line character. | \\n                  | STRING              | Yes      | No      |
-| mustache.enabled       | If this parameter is set to `true`, then mustache mapping gets enabled forcustom text mapping.                                                                                                                                                                                | false                | BOOL                | Yes      | No      |
+| event.grouping.enabled | If this parameter is set to `true`, events are grouped via a delimiter when multiple events are received. It is required to specify a value for the `delimiter` parameter when the value for this parameter is `true`.   | false                | BOOL | Yes      | No      |
+| delimiter              | This parameter specifies how events are separated when a grouped event is received. This must be a whole line and not a single character.    | \~\~\~\~\~\~\~\~\~\~ | STRING              | Yes      | No      |
+| new.line.character     | This attribute indicates the new line character of the event that is expected to be received. This is used mostly when communication between 2 types of operating systems is expected. For example, Linux uses `\n` whereas Windows uses `\r\n` as the end of line character. | \\n     | STRING              | Yes      | No      |
+| mustache.enabled       | If this parameter is set to `true`, then mustache mapping gets enabled for custom text mapping. | false    | BOOL                | Yes      | No      |
 
 ## Example 1
 
     CREATE SINK FooStream WITH (type='inMemory', topic='stock', map.type='text') (symbol string, price float, volume long);
 
-This query performs a default text input mapping. The expected output is
-as follows: symbol:"gdn", price:55.6, volume:100
+This query performs a default text input mapping. The expected output is:
+
+    symbol:"GDN",
+    price:55.6,
+    volume:100
 
 ## Example 2
 
     CREATE SINK FooStream WITH (type='inMemory', topic='stock', map.type='text', map.event.grouping.enabled='true') (symbol string, price float, volume long);
```

Comment:
Let's replace `inMemory` by `stream`.

Reply:
Should I just do this everywhere in the docs? Or is it on a case-by-case basis?

Reply:
Yes, this could be globally replaced.

Reply:
Then I shall do so in the JSON functions PR.